js/src/features/project/components/ProjectHeaderMenu.tsx

```diff
@@ -1,37 +1,72 @@
 import React from "react";
-import { Menu } from "antd";
+import { Menu, PageHeader, Tag, Typography } from "antd";
 import { Link, useLocation } from "react-router-dom";
+import { useSelector } from "react-redux";
+
+import { selectProject } from "../selectors";
+import { ProjectStatusTypeResponse } from "ProjectModels";
+import { chooseColor } from "../helpers";
+
+import "../project.scss";
+
+const { Paragraph } = Typography;
 
 type Props = {
   slug: string;
 };
 
 export const ProjectHeaderMenu: React.FC<Props> = ({ slug }) => {
```

Comment:
Looks like it is dumb component. Let try to send `project` via props.
Before send make sure that project is not empty (I hate endless `project?.` view)

Reply:
And you don't need `slug` prop coz it's already exists in `project`

Reply:
Done